std/php/Global.hx

```diff
@@ -1093,6 +1093,11 @@ extern class Global {
 	**/
 	static function mb_check_encoding(str:String = null, ?encoding:String):Bool;
 
+	/**
+		@see http://php.net/manual/en/function.mb-language.php
+	**/
+	static function mb_language(?language:String):EitherType<String,Bool>;
```

Comment:
This extern should be defined with overloads as in the PHP API reference because the return type depends on the set of arguments.

Reply:
Fixed :o)